src/components/custom-select/custom-select-items.html

```diff
@@ -1,5 +1,5 @@
 <% for(let i=0; i<list.length; i++) {%>
-  <li class="mdc-list-item" role="option" tabindex="0">
+  <li class="mdc-list-item" name="<%= list[i] %>" role="option" tabindex="0">
```

Comment:
why do you have `name` prop on `li` tag? if it is custom -- just use `data-name`

Reply:
because I need to select current tag/account. MDC select component gives this opportunity by `id` or `name`. I choose `name`